src/core/image_resizer.js

```diff
@@ -113,6 +130,11 @@ class ImageResizer {
     }
   }
 
+  static setOptions({ maxArea = -1, isChrome = false }) {
+    this.setMaxArea(maxArea);
+    this.#isChrome = isChrome;
+  }
```

Comment:
Similar to the previous comment.
```suggestion
  static setOptions(opts) {
    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
      throw new Error("Not implemented: setOptions");
    }
    this.setMaxArea(opts.maxArea ?? -1);
    this.#isChrome = opts.isChrome ?? false;
  }
```